data-prepper-plugins/s3-sink/src/main/java/org/opensearch/dataprepper/plugins/sink/s3/S3SinkConfig.java

```diff
@@ -43,8 +43,8 @@ public class S3SinkConfig {
     @JsonProperty("bucket_selector")
     private PluginModel bucketSelector;
 
-    @JsonProperty("predefined_object_metadata")
-    private PredefinedObjectMetadata predefinedObjectMetadata;
+    @JsonProperty("object_metadata")
```

Comment:
This configuration was already released in Data Prepper 2.9, so this is a breaking change.

Reply:
@dlvenable It is breaking change but no one is using.

Reply:
I can't say that is the case or not.